package.json

```diff
@@ -52,7 +52,7 @@
     "url": "https://github.com/mavenlink/brainstem-redux.git"
   },
   "license": "MIT",
-  "dependencies": {
+  "peerDependencies": {
     "brainstem-js": "^0.4.30",
     "jquery": "2.2.4",
     "redux": "^3.5.2",
```

Comment:
Should `redux-thunk` really be a peer dependency?

Reply:
Yes. Because the parent module will need thunk middleware to handle the async action creators from collection fetch, model fetch, model save, etc.